core/providers/githubcopilot/token_test.go

```
@@ -0,0 +1,795 @@
+
+	mux := http.NewServeMux()
+	mux.HandleFunc("/app/installations/", func(w http.ResponseWriter, r *http.Request) {
+		f.installationHits.Add(1)
+		if d := f.delay.Load(); d > 0 {
+			time.Sleep(time.Duration(d) * time.Millisecond)
+		}
+		status := int(f.installationStatus.Load())
+		w.Header().Set("Content-Type", "application/json")
+		w.WriteHeader(status)
+		if status != http.StatusCreated && status != http.StatusOK {
+			fmt.Fprint(w, `{"message":"upstream said no"}`)
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprint` is not checked

(errcheck)